.env

```diff
@@ -1 +1 @@
-REACT_APP_BASE_URL='https://18qv4o7xm9.execute-api.eu-west-1.amazonaws.com'
+REACT_APP_BASE_URL='https://skhcma77a3.execute-api.eu-west-1.amazonaws.com'
```

Comment:
your .env file is not in `.gitignore`.

Please include it there and remove this sensitive data from Git